sampleFileCacheConfig.yaml

```diff
@@ -1,7 +1,5 @@
 # Refer ./setup/baseConfig.yaml for full set of config parameters
 
-allow-other: true
```

Comment:
I think we should keep this in the sample configs because in most cases customers mount blobfuse2 as root(fstab for example) and try to access as regular user.